docs/docs/modules/model_io/llms/token_usage_tracking.ipynb

```diff
@@ -158,13 +168,95 @@
     "    print(f\"Total Cost (USD): ${cb.total_cost}\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "25d9ae710a452504",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## Bedrock"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "80ca77a3",
-   "metadata": {},
+   "execution_count": 4,
+   "id": "64f9ddd44b12773f",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-12-04T09:21:46.390766Z",
+     "start_time": "2023-12-04T09:21:46.195592Z"
+    },
+    "collapsed": false
+   },
    "outputs": [],
-   "source": []
+   "source": [
+    "import anthropic\n",
+    "from langchain.callbacks import get_bedrock_token_count_callback\n",
+    "from langchain.llms import Bedrock"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "697650deacad7df1",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-12-04T09:21:48.660152Z",
+     "start_time": "2023-12-04T09:21:48.551227Z"
+    },
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "llm = Bedrock(\n",
+    "    model_id=\"anthropic.claude-v2:1\",\n",
+    "    region_name=\"us-east-1\",\n",
+    "    model_kwargs={\"temperature\": 0.0},\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "4cc5cd6ed63b2e2a",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-12-04T09:22:51.525820Z",
+     "start_time": "2023-12-04T09:22:49.510715Z"
+    },
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " The square root of 4 is 2.\n",
+      " The square root of 4 is 2.\n",
+      "58 34 24\n"
+     ]
+    }
+   ],
+   "source": [
+    "with get_bedrock_token_count_callback() as cb:\n",
+    "    text = \"\\n\\nHuman: What is the square root of 4?\\n\\nAssistant: \"\n",
+    "    answer = llm(text)\n",
+    "    print(answer)  # check if invocations return same response\n",
+    "    answer = llm(text)\n",
+    "    print(answer)  # check if invocations return same response\n",
+    "\n",
+    "    total_tokens = cb.total_tokens\n",
+    "    print(cb.total_tokens, cb.prompt_tokens, cb.completion_tokens)\n",
```

Comment:
Currently, this is just printing numbers which doesn't have any info on the counts. Updating this to show the counts and their purpose would be useful.
```suggestion
    "    print(cb)\n",
```